config/routes.rb

```diff
@@ -6,6 +6,7 @@
 
   namespace :admin do
     root to: 'pages#dashboard'
+    resources :kudos
```

Comment:
as this is RD action you could limit it's routes by `, only: [:index, :other_method]`

Reply:
introduced in the next commit. Thanks! ;)